File: backend-gateway/src/main/java/cc/jktu/apps/gateway/auth/StpInterfaceImpl.java
package cc.jktu.apps.gateway.auth;

import cn.dev33.satoken.stp.StpInterface;
import org.springframework.stereotype.Component;

import java.util.List;

@Component
public class StpInterfaceImpl implements StpInterface {

    @Override
    public List<String> getPermissionList(Object loginId, String loginType) {
        return List.of();
    }

    @Override
    public List<String> getRoleList(Object loginId, String loginType) {
        return List.of();
    }

}

File: backend-gateway/src/main/kotlin/cc/jktu/apps/gateway/auth/StpInterfaceImpl.kt
package cc.jktu.apps.gateway.auth

import cn.dev33.satoken.stp.StpInterface
import org.springframework.stereotype.Component

@Component
class StpInterfaceImpl : StpInterface {

    override fun getPermissionList(loginId: Any, loginType: String): List<String> {
        return listOf()
    }

    override fun getRoleList(loginId: Any, loginType: String): List<String> {
        return listOf()
    }
}